CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
-### 1.9.0 (Next)
+### 2.0.0 (Next)
```

Comment:
Increment the version in version.rb as well, pls.

Reply:
Incremented by [9d75263](https://github.com/ruby-grape/grape/pull/2361/commits/9d7526342a1cc7ef327f90b7fcdacc8126dcc574).